conda_build/post.py

```diff
@@ -395,7 +395,9 @@ def osx_ch_link(path, link_dict, host_prefix, build_prefix, files):
     return ret
 
 
-def mk_relative_osx(path, host_prefix, build_prefix, files, rpaths=('lib',)):
+def mk_relative_osx(path, host_prefix, m, files, rpaths=('lib',)):
+    base_prefix = m.config.build_folder
```

Comment:
@isuruf, I've changed `dirname(host_prefix)` to `m.config.build_folder` so this is more explicit and hopefully more robust in case someone ever wanted to change the depth of folders under `build_folder`.
We could also add
```suggestion
    base_prefix = m.config.build_folder
    assert base_prefix == dirname(host_prefix)
```
to guard for regressions if such change would happen. (I just didn't include the assertion right away because I couldn't remember/didn't look at how/if `assert`s are used within `conda-build` -- but there are probably no hard conventions :D.)

Reply:
Thanks for adding that. I'm fine with either way about the assertion.